grpc/src/main/java/com/linecorp/armeria/server/grpc/ArmeriaCoroutineContextInterceptor.java

```diff
@@ -37,13 +76,27 @@ final class ArmeriaCoroutineContextInterceptor extends CoroutineContextServerInt
     @Override
     public CoroutineContext coroutineContext(ServerCall<?, ?> serverCall, Metadata metadata) {
         final ServiceRequestContext ctx = ServiceRequestContext.current();
-        final ArmeriaRequestCoroutineContext coroutineContext = new ArmeriaRequestCoroutineContext(ctx);
-        final ScheduledExecutorService executor;
-        if (useBlockingTaskExecutor) {
-            executor = ctx.blockingTaskExecutor().withoutContext();
+        CoroutineContext coroutineContext = new ArmeriaRequestCoroutineContext(ctx);
+        if (PROVIDE_METHOD != null && !COROUTINE_CONTEXT_PROVIDERS.isEmpty()) {
```

Comment:
Question)
Is it possible for us to define a default dispatcher, and then users can override the dispatcher by adjusting the ordering?

i.e.
```
val ctx = ExecutorsKt.from(executor).plus(coroutineContext)
for (Object provider : COROUTINE_CONTEXT_PROVIDERS) {
  coroutineContext = coroutineContext.plus((CoroutineContext) PROVIDE_METHOD.invoke(provider, ctx));
...
```

Reply:
I'm also wondering if there's a case that the order of provider matters.

Reply:
Good point. Let me try to test it.

Reply:
I realized that it was an intentional behavior to allow setting an empty dispatcher to a coroutine context. A coroutine dispatcher specified by `CoroutineContextInterceptor` overrides the context dispatcher in a coroutine stub.
https://github.com/grpc/grpc-kotlin/blob/5f68652e324579881ce5cc8900f16560a59f5fa8/stub/src/main/java/io/grpc/kotlin/ServerCalls.kt#L202-L204

Reply:
I see. I was wondering if this extension would still be useful for users who want to add additional `CoroutineContext`s but still use armeria's `eventLoop` as the default dispatcher.

I guess since `CommonPools.eventLoop()` is publicly available, users may add it themselves if they want it.
I guess it's fine to have this as the default behavior then.

Reply:
As `ArmeriaCoroutineContextInteceptor` is an internal API so I wrote the behavior in the Kotlin integration documentation. automatically injected https://github.com/line/armeria/pull/4866/files#diff-1ad174a00b4c2d6337f55b84b8f237dd7d9a2cc6be3ac2f41776a7819a23f99bR209